### PR TITLE
First attempt Homework5 for marking

### PR DESCRIPTION
**Pull request by emb3rlight**

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter1/ArrayOperations.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927151&bid=16300757)

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
-
         /// <summary>
         /// Reverse the array elements, first being last and so on.
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (IsNullorEmpty(array)) return;
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Codacy Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927151)

**Comment by Almantask** [comment]
Please fix the broken tests. You cannot just reuse the function of InsertAt as is, there are edge conditions for length 0, null, etc.

**Comment by Almantask** [comment]
Expired
